ld/armv7m7-imxrt106x.ldt

```diff
@@ -28,17 +28,21 @@
 /* FlexRAM configuration */
 #if defined(CUSTOM_FLEXRAM_CONFIG)
 FLEXRAM_CONFIG = CUSTOM_FLEXRAM_CONFIG;
+FLEXRAM_ITCM_BANKS = CUSTOM_FLEXRAM_ITCM_BANKS;
+FLEXRAM_DTCM_BANKS = CUSTOM_FLEXRAM_DTCM_BANKS;
 #else
 FLEXRAM_CONFIG = 0xaaaaabff;
+FLEXRAM_ITCM_BANKS = 5;
+FLEXRAM_DTCM_BANKS = 11;
 #endif
 
 /* Memory map setup */
 MEMORY
 {
 	/* TODO: use FLEXRAM_CONFIG value to setup ocram/itcm/dtcm partitioning (*32k) */
-	m_itcm    (rwx) : ORIGIN = 0x00000000, LENGTH = 5 * 32k
+	m_itcm    (rwx) : ORIGIN = 0x00000000, LENGTH = FLEXRAM_ITCM_BANKS * 32k
 	m_itext   (rwx) : ORIGIN = 0x00026800, LENGTH = 6 * 1k
-	m_dtcm    (rw)  : ORIGIN = 0x20002000, LENGTH = 11 * 32k
+	m_dtcm    (rw)  : ORIGIN = 0x20002000, LENGTH = FLEXRAM_DTCM_BANKS * 32k
 	m_ocram   (rwx) : ORIGIN = 0x20200000, LENGTH = 0 * 32k
```

Comment:
`m_ocram` is part of FlexRAM configuration, so it should be included in custom config.

Reply:
No idea why OCRAM length is set to 0. To be done later.

Reply:
Because flexram can be assigned 3-way: to itcm, to dtcm or to ocram. Our default configuration assign none to ocram, that's why it's 0

Reply:
Understood. There's no need to assign it to OCRAM now.